sass/_SimpleFilter.scss

```diff
@@ -0,0 +1,153 @@
+@import "bourbon/bourbon";
+@import "Variables";
+@import "../bin/sass/sprites";
+
+.coveo-filter-header-wrapper {
+  width: 100%;
+  margin:0 15px;
+  padding-left: 15px;
+  max-width: 1200px;
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+  display: block;
+  @include flex-wrap(nowrap);
+  @include justify-content(flex-start);
+}
+.coveo-dropdown-background {
+  opacity: 0.9;
+}
+.CoveoSimpleFilter {
+  display: inline-block;
+  position: relative;
+  border: 1px #dadada solid;
```

Comment:
All colors should use one available in `_Variables`. We should not be seeing color hex in any component css.

There's also available variable for default borders.